### ISSUE TITLE
Import xls issue (bug?)

### ISSUE THREAD

**Issue by chrismclarke** [africanmathsinitiative/R-Instat#2690]
Hi guys,

I'm using version 0.2 and tried reading in the following xlsx file but it failed - no tabs listed and preview window shows 'cannot import this file with the current options...'. 
[Varietal_diversitybaseline.xlsx](https://github.com/africanmathsinitiative/Instat/files/772129/Varietal_diversitybaseline.xlsx)

**Comment by dannyparsons** [comment]
I'm getting the same error with this file. Will investigate the issue. Thanks.

**Comment by dannyparsons** [comment]
Hi Chris, this seems to be an issue with the import function we in R for xlsx.
It may be related to this issue: http://stackoverflow.com/questions/36180661/r-libraryopenxlsx-error-workbook-has-no-worksheets

Although I got the same error as you the first time I tried this, when I opened the file in Excel, made a change and saved it, it then imported fine into R-Instat. Are there any special settings you know of on the file?
